lib/reporter.js

```diff
@@ -1,549 +1,607 @@
-'use strict';
-
-var jsonFile = require('jsonfile');
-var _ = require('lodash');
-var fs = require('fs-extra');
-var path = require('path');
-var jsonDir = require('./jsonDir');
-var open = require('open');
+// 'use strict';
 const chalk = require('chalk');
+const jsonFile = require('jsonfile');
+const _ = require('lodash');
+const fs = require('fs-extra');
+const path = require('path');
+const jsonDir = require('./jsonDir');
 const emoji = require('node-emoji');
-var searchFileUp = require('./searchFileUp');
-var hierarchyReporter = require('./hierarchyReporter');
-const guid = require('uuid/v4');
-const isBase64 = (data) => /^(?:[A-Z0-9+\/]{4})*(?:[A-Z0-9+\/]{2}==|[A-Z0-9+\/]{3}=|[A-Z0-9+\/]{4})$/i.test(data);
-
-var generateReport = function (options) {
-    var featureOutput = jsonFile.readFileSync(options.jsonFile);
-    var packageJsonPath = searchFileUp('package.json');
-    var packageJson = {};
-    try {
-        packageJson = packageJsonPath && jsonFile.readFileSync(packageJsonPath, 'utf8');
-    } catch (err) {
-        console.warn('No package.json file found in: ' + packageJsonPath + ', using default name and version.');
-        packageJson.name = 'default';
-        packageJson.version = '0.0.0';
+const searchFileUp = require('./searchFileUp');
+const hierarchyReporter = require('./hierarchyReporter');
+const guid = require('uuid').v4;
+const open = require('open');
+let isBase64 = (data) => /^(?:[A-Z0-9+/]{4})*(?:[A-Z0-9+/]{2}==|[A-Z0-9+/]{3}=|[A-Z0-9+/]{4})$/i.test(data);
+
+let stageRerunCount = getStageReRunCount();
+
+const generateReport = function (options) {
+  let featureOutput = jsonFile.readFileSync(options.jsonFile);
+  let packageJsonPath = searchFileUp('package.json');
+  let packageJson = {};
+  try {
+    packageJson = packageJsonPath && jsonFile.readFileSync(packageJsonPath, 'utf8');
+  } catch (err) {
+    console.warn('No package.json file found in: ' + packageJsonPath + ', using default name and version.');
+    packageJson.name = 'default';
+    packageJson.version = '0.0.0';
+  }
+
+  const sanitize = function (name, find) {
```

Comment:
That's another feature to list in the PR description and in the changelog.
Some projects might have their own sanitizers implemented and that feature will require to take some cleanup on the projects.